dso-l2/src/main/java/com/tc/objectserver/handler/ProcessTransactionHandler.java

```diff
@@ -106,6 +108,12 @@ public synchronized void reconnectComplete() {
     public void handleEvent(TCMessage context) throws EventHandlerException {
       NodeID destinationID = context.getDestinationNodeID();
       invokeReturn.remove((ClientID)destinationID, context);
+      if(context instanceof VoltronEntityMultiResponse) {
```

Comment:
Life cycle requests responses are not batched and they are being handled using AbstractServerEntityRequestResponse.received()